src/components/TaskListItem/TaskListItem.jsx

```diff
@@ -0,0 +1,218 @@
+                        {focusedTask ? < KeyboardArrowDownIcon
+                            sx={{ fontSize: 25, color: "icons.secondary" }} /> : <KeyboardArrowRightOutlinedIcon
+                            sx={{ fontSize: 25, color: "icons.secondary" }} />}
+                    </IconButton>
+                </Box>
+
+                <CSSTransition
+                    in={focusedTask}
+                    timeout={500}
+                    classNames="todo-additional-container"
+                    unmountOnExit>
+                    <Box
+                        sx={{
+                            bgcolor: 'background.paper',
+                            display: 'flex',
+                            gap: '15px',
+                            height: '57px',
+                            paddingLeft: '29px',
+                            boxShadow: '0px 4px 8px -3px rgba(0, 0, 0, 0.25)'
+                        }}
+                        onKeyDown={(e) => {
+                            if (e.key === 'Enter') {
+                                dispatch(setTheDateOfPerfomingTheTask({ name: text, date: dayjs(date).format('DD.MM.YYYY') }))
+                                request(`http://localhost:3001/tasks/${idOfTheTask}`, 'PUT', JSON.stringify({
+                                    ...tasks[indexOfTheTask],
+                                    date: dayjs(date).format('DD.MM.YYYY')
+                                }))
+                                setShowCalendar(false)
+                                setFocusedTask(false)
+                            }
+                        }}
+                    >
+
+                        {showCalendar ? (<LocalizationProvider dateAdapter={AdapterDayjs}>
+                            <DateField
+                                autoFocus={true}
+                                onChange={(newValue) => {
+                                    setDate(newValue)
+                                }}
+                                format='DD.MM.YYYY'
+                                value={date}
+                                label="Set date"
+                            />
+                        </LocalizationProvider>) : (<IconButton
+                            onClick={() => { setShowCalendar(true) }}>
+                            <DateRangeOutlinedIcon
+                                sx={{ fontSize: 25, color: 'icons.primary' }} />
+                        </IconButton>)}
+                        <IconButton onClick={() => {
+                            dispatch(setAsImportantTask(text));
+                            request(`http://localhost:3001/tasks/${idOfTheTask}`, 'PUT', JSON.stringify({
+                                ...tasks[indexOfTheTask],
+                                important: !tasks[indexOfTheTask].important
+                            }))
+                        }}>
+                            {indexOfTheTask !== null && tasks[indexOfTheTask] && tasks[indexOfTheTask].important ?
```

Comment:
please create a variable
```suggestion
                            {showSmth ?
```
const showSmth = indexOfTheTask !== null && tasks[indexOfTheTask] && tasks[indexOfTheTask].important;

Reply:
done